bsip-0022.md

```diff
@@ -37,34 +37,202 @@ Currently within Bitshares DPOS system, an user's votes for committee/witness/pr
+
+### <div id="peerplays-vote-decay" /> Vote Decay on Peerplays
+
+A [proposal for Peerplays](https://github.com/peerplays-network/pips/blob/master/pip-0002.md) introduced vote decay as an element of its Gamified Proof of Stake (GPOS).  Voters receive rewards from a portion of the operation fees and "rakes" that are collected by the blockchain.  The reward is contingent on [the amount of stake that is vested by the voter](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L1072-1080), and [the age of last casted vote by the voter](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L726-791).  The age of the last casted vote [also decays the voting power](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L1446-1447) for all referendums on the platform.
+
+
 # Specifications
 
-## Committee controlled variables
+## <div id="referendum-categories" /> Referendum Categories
+
+BitShares has four categories of refereundums that core token holders may vote on: witnesses, committee members, worker proposals, and proxy delegation.  Prior to this proposal, any vote within any category applies until changed or until the proxy assignment had been assigned.  The removal of proxy assignment re-instates a core token holders' prior voting selection for witnesses, commitee members.
+
+This proposal shall retain the ability of every core token holder to vote on all referendum categories and the optional selection of a single proxy.
+
+Vote for three of the referendum categories shall be "decayable": witnesses, committee members, and proxy delegation.  Votes for worker proposals [shall continue to be tabulated as they were prior to this proposal](#out-of-scope) because they are generally fixed in length.
```

Comment:
~length~ duration

Reply:
The intent is to retain the current scheme of vote tallying for worker proposals: 

- If a token holder has not selected proxy then their stake is assigned to whatever worker proposal is selected by the token holder _at the time of vote tallying_.

- If a token holder has selected a proxy then their stake is assigned to whatever worker proposal is selected by the proxy _at the time of vote tallying_.

Reply:
What I mean is - you say that proxy delegation is decayable but worker votes are not. If I have set a proxy and the proxy votes for workers, do my votes for the worker count in full or as per decay rules?